biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -1,3 +1,18 @@
+/**
+ * @file defines the @class BioBattery which describes the battery class which is used for storing energy.
```

Comment:
nit: a bit confusing having "which" twice in the same sentence, mind clarifying this first sentence?